--- ./acinclude.m4.orig	2016-12-16 16:45:27.000000000 +0100
+++ ./acinclude.m4	2021-01-06 10:51:04.605812682 +0100
@@ -710,6 +710,7 @@
 
 #if ! defined (__cplusplus)
 #include <stdlib.h>
+#include <string.h>
 void
 lshift_com (rp, up, n, cnt)
   unsigned long *rp;
@@ -742,8 +743,9 @@
   for (i = 0; i < 88 + 1; i++)
     a[i] = ~0L;
   r = malloc (10000 * sizeof (unsigned long));
+  memset (r, 0x3F, 10000 * sizeof (unsigned long));
   r2 = r;
-  for (i = 0; i < 528; i += 23)
+  for (i = 0; i < 528; i += 22)
     {
       lshift_com (r2, a,
 		  i / (8 * sizeof (unsigned long)) + 1,
