diff -up linux-5.6.15/drivers/net/wireless/realtek/rtw88/pci.c.orig linux-5.6.15/drivers/net/wireless/realtek/rtw88/pci.c
--- linux-5.6.15/drivers/net/wireless/realtek/rtw88/pci.c.orig	2020-05-27 17:48:31.000000000 +0200
+++ linux-5.6.15/drivers/net/wireless/realtek/rtw88/pci.c	2020-10-16 17:00:25.093957406 +0200
@@ -1515,6 +1515,7 @@ static const struct pci_device_id rtw_pc
 #endif
 #ifdef CONFIG_RTW88_8822CE
 	{ RTK_PCI_DEVICE(PCI_VENDOR_ID_REALTEK, 0xC822, rtw8822c_hw_spec) },
+	{ RTK_PCI_DEVICE(PCI_VENDOR_ID_REALTEK, 0xC82f, rtw8822c_hw_spec) },
 #endif
 	{},
 };
